provisioning/datastore.yml

```diff
@@ -21,7 +21,12 @@
   become: true
 
 - name: set default group write ACLs on /srv/datastore
-  command: /usr/bin/setfacl -m d:g::rwx /srv/datastore
```

Comment:
We should carefully check if this works. I didn't use the acl module because it didn't work correctly. The settings below suggest that every flag is accepted as a parameter to the module, so it probably works now. But we should check!